<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TVWidgetClass</class>
 <widget class="QWidget" name="TVWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>434</height>
   </rect>
  </property>
  <widget class="QFrame" name="tvBarFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>388</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>27</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>8</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="topMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="tvIconLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>14</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="moodbox.qrc">:/MoodBox/Resources/tv_black.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>3</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="MoodBox::ElidedLabel" name="chatLabel"/>
    </item>
    <item>
     <widget class="QLabel" name="messageDateLabel">
      <property name="accessibleName">
       <string>barGrayLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="MoodBox::ElidedPushButton" name="userNameButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>true</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="sentLabel">
      <property name="accessibleName">
       <string>barGrayLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="replyRightArrowButton">
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>true</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="accessibleName">
       <string>redButton</string>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Reply</string>
      </property>
      <property name="icon">
       <iconset resource="moodbox.qrc">
        <normaloff>:/MoodBox/Resources/cl_right_arrow_red.png</normaloff>:/MoodBox/Resources/cl_right_arrow_red.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="MoodBox::TVPreviewLabel" name="previewLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>27</y>
     <width>388</width>
     <height>240</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>388</width>
     <height>240</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>388</width>
     <height>240</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="tvBarShadow">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>27</y>
     <width>388</width>
     <height>4</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>388</width>
     <height>4</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>388</width>
     <height>4</height>
    </size>
   </property>
   <property name="accessibleName">
    <string>transBorder</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QFrame" name="transBorder_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>2</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>2</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>2</height>
    </size>
   </property>
   <property name="accessibleName">
    <string>transBorder</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QFrame" name="transBorder">
   <property name="geometry">
    <rect>
     <x>386</x>
     <y>2</y>
     <width>2</width>
     <height>263</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>2</width>
     <height>263</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>2</width>
     <height>263</height>
    </size>
   </property>
   <property name="accessibleName">
    <string>transBorder</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
  </widget>
  <widget class="QFrame" name="transBorder_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>2</y>
     <width>2</width>
     <height>263</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>2</width>
     <height>263</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>2</width>
     <height>263</height>
    </size>
   </property>
   <property name="accessibleName">
    <string>transBorder</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QFrame" name="transBorder_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>265</y>
     <width>401</width>
     <height>2</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>2</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>2</height>
    </size>
   </property>
   <property name="accessibleName">
    <string>transBorder</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="obsceneButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>165</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>ObsceneButtonHint</string>
   </property>
   <property name="accessibleName">
    <string>obscenePushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteMessageButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>91</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>DeleteMessageButtonHint</string>
   </property>
   <property name="accessibleName">
    <string>deleteMessagePushButton</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>MoodBox::ElidedLabel</class>
   <extends>QLabel</extends>
   <header>elidedlabel.h</header>
  </customwidget>
  <customwidget>
   <class>MoodBox::ElidedPushButton</class>
   <extends>QPushButton</extends>
   <header>elidedpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>MoodBox::TVPreviewLabel</class>
   <extends>QLabel</extends>
   <header>tvpreviewlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="moodbox.qrc"/>
 </resources>
 <connections/>
</ui>
